packages/arb-token-bridge-ui/tests/e2e/specs/switchNetworks.cy.ts

```diff
@@ -0,0 +1,128 @@
+      cy.findByRole('button', { name: /From: Goerli/i }).should('be.visible')
+      cy.findByRole('button', { name: /To: Arbitrum Goerli/i }).should(
+        'be.visible'
+      )
+    })
+
+    context('Test Networks dropdown in Nav bar', () => {
+      it('should show and open the networks dropdown', () => {
+        cy.findByRole('button', { name: /Selected Network : /i })
+          .should('be.visible')
+          .click({ scrollBehavior: false })
+
+        cy.findByRole('button', { name: /Switch to Arbitrum One/i }).should(
+          'be.visible'
+        )
+
+        //close the dropdown
+        cy.findByRole('button', { name: /Selected Network : /i })
+          .should('be.visible')
+          .click({ scrollBehavior: false })
+      })
+
+      it('should change network to Arbitrum One successfully', () => {
+        cy.findByRole('button', { name: /Selected Network : /i })
+          .should('be.visible')
+          .click({ scrollBehavior: false })
+
+        cy.findByRole('button', { name: /Switch to Arbitrum One/i })
+          .should('be.visible')
+          .click({ scrollBehavior: false })
+
+        cy.allowMetamaskToAddAndSwitchNetwork().then(() => {
+          cy.findByRole('button', {
+            name: /Selected Network : Arbitrum One/i
+          }).should('be.visible')
+        })
+      })
+
+      it('should change network to Arbitrum Nova successfully', () => {
+        cy.findByRole('button', { name: /Selected Network : /i })
+          .should('be.visible')
+          .click({ scrollBehavior: false })
+
+        cy.findByRole('button', { name: /Switch to Arbitrum Nova/i }).click({
+          scrollBehavior: false
+        })
+
+        cy.allowMetamaskToAddAndSwitchNetwork().then(() => {
+          cy.findByRole('button', {
+            name: /Selected Network : Arbitrum Nova/i
+          }).should('be.visible')
+        })
+      })
+
+      it('should change network to Ethereum mainnet successfully', () => {
+        cy.findByRole('button', { name: /Selected Network : /i })
+          .should('be.visible')
+          .click({ scrollBehavior: false })
+
+        cy.findByRole('button', { name: /Switch to Mainnet/i }).click({
+          scrollBehavior: false
+        })
+
+        cy.allowMetamaskToSwitchNetwork().then(() => {
+          cy.findByRole('button', {
+            name: /Selected Network : Mainnet/i
+          }).should('be.visible')
+        })
+      })
+    })
+
+    context('Test Networks list in Wrong Network UI', () => {
+      it('should show wrong network UI', () => {
+        cy.changeMetamaskNetwork('kovan').then(() => {
+          cy.findByText(/Oops! You’re connected to a wrong network./i).should(
+            'be.visible'
+          )
+        })
+      })
+
+      it('should allow Network change from wrong network UI list', () => {
+        cy.changeMetamaskNetwork('kovan').then(() => {
+          cy.findByRole('button', { name: /Switch to Arbitrum One/i })
```

Comment:
if the user is on an unsupported testnet, should we direct them to goerli instead?